dist/tools/drone-scripts/build_and_test.sh

```diff
@@ -16,9 +16,16 @@ function exec_build_func {
     fi
 
     echo "Begin group ${group}"
+    # spawn new directory for the group
+    mkdir -p "build_tmp/${group}"
+    rsync -a --exclude="build_tmp" . "build_tmp/${group}"
```

Comment:
`cp -al` (hard linking) uses no additional hard disk space and takes less time to perform, I am unsure however on how to do the exclusion.